Networking/Networking/Remote/StripeRemote.swift

```diff
@@ -35,7 +35,24 @@ public class StripeRemote: Remote {
 
     /// TODO captureOrderPayment(for siteID: Int64,...)
 
-    /// TODO fetchOrderCustomer(for siteID: Int64,...)
+    /// Creates a (or returns an existing) Stripe Connect customer for an order. See https://stripe.com/docs/api/customers/create
+    /// Updates the order meta with the Customer for us.
+    /// Also note that the JSON returned by the WCPay endpoint is an abridged copy of Stripe's response.
+    /// - Parameters:
+    ///   - siteID: Site for which we'll create (or simply return) the customer.
+    ///   - orderID: Order for which we'll create (or simply return) the customer.
+    ///   - completion: Closure to be run on completion.
+    public func fetchOrderCustomer(for siteID: Int64,
+                               orderID: Int64,
+                               completion: @escaping (Result<Customer, Error>) -> Void) {
+        let path = "\(Path.orders)/\(orderID)/\(Path.createCustomer)"
```

Comment:
I confirmed the path, just to be sure :) https://github.com/woocommerce/woocommerce-gateway-stripe/blob/15f3006c8950409d128f514f6e4effc2668e251f/includes/admin/class-wc-rest-stripe-orders-controller.php#L42